Update PR template to specify desired git commit messages (#20048)
kubernetes/website: .github/PULL_REQUEST_TEMPLATE.md

 your pull request. The description should explain what will change,
 and why.

 PLEASE title the FIRST commit appropriately, so that when the pull
 request is merged and squashed, the commit message makes sense.

> 

Reviewer: @onlydole this repo now uses merge-method-merge as the default (previously it was merge-method-squash).

Is this advice still right? I agree about the message, but it might imply that the repo still uses merge-method-squash.

Reviewer: ```suggestion
 PLEASE title the FIRST commit appropriately, so that if you squash all
 your commits into one, the combined commit message makes sense.
```
?
